fs/expose/dokan/__init__.py

```diff
@@ -76,7 +85,9 @@
 from fs.wrapfs import WrapFS
 
 try:
-    import libdokan
+    if six.PY2: import libdokan
+    elif six.PY3: from . import libdokan
```

Comment:
Couldn't this whole conditional-chunk be eliminated by just adding `from __future__ import absolute_import` to the top?